eks-worker-al2.json

```diff
@@ -192,12 +194,20 @@
     {
       "type": "manifest",
       "output": "manifest.json",
-      "strip_path": true
+      "strip_path": true,
+      "custom_data": {
```

Comment:
Perhaps just calls this `metadata`?

Reply:
https://www.packer.io/docs/post-processors/manifest#custom_data`custom_data` you mean? That's a property of the manifest post-processor:

Reply:
Ok, NVM. Wasn't sure if you were defining this or not.